### ISSUE TITLE
Mapped Network Drives between Windows PCs

### ISSUE THREAD

**Issue by ms52538** [BiglySoftware/BiglyBT#1315]
Hey gents, hope all is well.

Question: on the latest beta I began having to run the client on a second PC to a tracker, within VPN. 

It appears that when I attempt to initially save a newly downloaded torrent and have it map to an existing file on my primary PC (Windows 10, latest build, mapped network drives) - I receive an error in the attempt. However, if I save the torrent and point to my local fold on the second PC, it saves fine as expected and I can then tell it to Find Existing Files to Match through my mapped network drive from the second computer to my primary PC. In that step, the torrent on my second PC finds the corresponding file on my primary PC, just fine. 

It just struck me as odd that one does not work (as you think it would) and the other does work (when you think it wont because the first way failed). I may be simply missing something in the configuration of the client? Not sure, but thought I'd toss it out at you.  Thanks! M

**Comment by parg** [comment]
Hmm, within BiglyBT we don't know much about network drives, we just let the system dialogs browse for files/folders, get the paths they return and then effectively pass them back to the OS when opening files etc. What is the error message you are getting when it fails?

**Comment by ms52538** [comment]
First, I apologize for wasting your time - I should have researched this a bit more before posting.
It turned out to be a permissions issue on the source/file host PC. 
What threw me was the Find Files feature works - so I ruled out a permission issue (I was not getting an access denied error when performing a Find Existing Files function). But after repeating initial setting up the torrent to download, map to the networked source/file host PC, matched the existing file name with the torrent name being downloaded, and bam, no magic happened but I looked more closely at the error message this time. I caught the Access Denied error message being thrown from the torrent, which meant the Originating PC could not create/write to the source/file host PC. Once I changed that permissions on the source/file host PC so the user name explicitly existed with admin/read/write permission (using a cloud-based account with Microsoft), the problem was resolved.
